### PR TITLE
 [XHarness] Add bcl tests for System.IO.Compression.FileSystem.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

File: tests/Makefile [hunk 1]
```diff
@@ -12,7 +12,25 @@ MTOUCH=$(IOS_DESTDIR)/$(MONOTOUCH_PREFIX)/bin/mtouch
 UNIT_SERVER_DIR=$(TOUCH_UNIT_PATH)/Touch.Server
 UNIT_SERVER=$(UNIT_SERVER_DIR)/bin/Debug/Touch.Server.exe
 TEST_SUITES=monotouch-test link\ sdk link\ all dont\ link framework-test mini
-BCL_TEST_SUITES=mscorlib System System.Core System.Data System.Net.Http System.Numerics System.Runtime.Serialization System.Transactions System.Web.Services System.Xml System.Xml.Linq Mono.Security System.ComponentModel.DataAnnotations System.Json System.ServiceModel.Web Mono.Data.Sqlite Mono.Data.Tds System.IO.Compression
+BCL_TEST_SUITES=mscorlib \
```
Comment (dalexsoto) [inline tests/Makefile]: ahh nice! more readable 👍

File: tests/bcl-test/System.IO.Compression.FileSystem/Info-mac.plist [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDisplayName</key>
+	<string>corelibtests</string>
```
Comment (rolfbjarne) [inline tests/bcl-test/System.IO.Compression.FileSystem/Info-mac.plist]: It's minor, but it's a little bit less confusing when looking at installed apps if the name matches the actual tests :smile: IOW maybe use something like `System.IO.Compression.FileSystemTests` as the `CFBundleDisplayName`.

Comment (mandel-macaque) [reply tests/bcl-test/System.IO.Compression.FileSystem/Info-mac.plist]: Sure, makes sense. Will update and merge the branch.

**Comment by mandel-macaque** [comment]
Looks like a failure not related to the branch, rebuilding to double check.

**Comment by mandel-macaque** [comment]
build

**Comment by spouliot** [comment]
@mandel-macaque please always investigate bot failures and find the related (or file a new) bug. Only kick a new build if the tests ran did not cover your change.

**Comment by mandel-macaque** [comment]
Failing build last time was related -> https://bugzilla.xamarin.com/show_bug.cgi?id=60251

**Pull request merged**